nix/config.nix

```diff
@@ -0,0 +1,8 @@
+/*
+  This file contains default values for parameters used in nix derivations.
+  Set a value here if you want to share a nix value across all of the repos
+  which use this repo as a git submodule.
```

Comment:
Add back once we move this to common submodule
```suggestion
```

Reply:
Oh, I forgot about the comment. Removed the last 2 lines.